project/Build.scala

```diff
@@ -269,6 +270,16 @@ object ScaldingBuild extends Build {
       "org.apache.hadoop" % "hadoop-core" % hadoopVersion % "provided",
       "org.slf4j" % "slf4j-api" % slf4jVersion,
       "org.slf4j" % "slf4j-log4j12" % slf4jVersion % "provided"
+    ) ++ (
+      if(isScala210x(scalaVersion.value)) {
+        Seq(
+          "com.chuusai" % "shapeless_2.10.4" % shapelessVersion
```

Comment:
Shapeless 2.1.0 has full cross-versions for 2.10 ([e.g for 2.10.5](https://repo1.maven.org/maven2/com/chuusai/shapeless_2.10.5/2.1.0/)), so it'd be better to use this:

    "com.chuusai" %% "shapeless" % "2.1.0" cross CrossVersion.full

So that we get the 2.10.5 version when Scalding updates to 2.10.5.

(I don't remember whether you need `%%` for full cross-versioning—I think either will work, and I find it a little clearer with `%%`.)